examples/java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

Comment:
The latest gradle version is 7.5.1

Reply:
We aren't actually using gradle in this project right now, I just want a place to show the right code/version, so I'm less worried that this stays pegged to the latest version.